Gemfile

```diff
@@ -48,7 +48,7 @@ gem 'laa-criminal-applications-datastore-api-client',
     require: 'datastore_api'
 
 gem 'laa-criminal-legal-aid-schemas',
-    github: 'ministryofjustice/laa-criminal-legal-aid-schemas', tag: 'v1.0.46'
+    github: 'ministryofjustice/laa-criminal-legal-aid-schemas', ref: 'acf042811accfdd413b4aaf8f9ec810c182c2fba'
```

Comment:
Is the schema PR ready to review yet @ripan? It would be good to get that in first so you can use the tag here